runtime/include/chpl-comm-compiler-llvm-support.h

```diff
@@ -40,15 +40,15 @@ void chpl_gen_comm_get_ctl(void *dst_addr, wide_ptr_t src, int64_t n, int64_t ct
 {
   c_nodeid_t src_node = chpl_wide_ptr_get_node(src);
   void* src_addr = chpl_wide_ptr_get_address(src);
-  chpl_gen_comm_get(dst_addr, src_node, src_addr, sizeof(uint8_t)*n, CHPL_TYPE_uint8_t, -1, 0);
+  chpl_gen_comm_get(dst_addr, src_node, src_addr, sizeof(uint8_t)*n, CHPL_TYPE_uint8_t, CHPL_UNKNOWN_COMM_ID, -1, 0);
```

Comment:
I'd prefer CHPL_COMM_ID_UNKNOWN or CHPL_COMM_UNKNOWN_ID here, just to group this symbol with the rest of the (case insensitive) chpl_comm_* ones.